src/main/java/seedu/address/logic/commands/EditCategory.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.category.Category;
+import seedu.address.model.category.UserDefinedCategory;
+
+
+/**
+ * Edits a category in the ExpenseTracker
+ */
+public class EditCategory extends Command {
+    public static final String COMMAND_WORD = "ecat";
+
+    public static final String MESSAGE_NO_EXISTING_CATEGORY = "There is no such category!";
+
+    public static final String MESSAGE_DEFAULT_CATEGORY = "The MISC category can not be edited!";
+    private final Index targetIndex;
+
+    private final String newCategoryName;
```

Comment:
follows command structure

Reply:
Should we edit the command for categories to acat - add category, dcat to delete category to follow the formatting of ecat and lcat?